src/modules/30-secrets/components/TextReference/TextReference.tsx

```diff
@@ -44,11 +44,11 @@ interface TextReferenceProps {
 }
 
 interface FormikTextReference extends TextReferenceProps {
-  formik: FormikContext<any>
+  formik: FormikContextType<any>
   tooltipProps?: DataTooltipInterface
 }
 
-const errorCheck = (name: string, formik?: FormikContext<any>) =>
+const errorCheck = (name: string, formik?: FormikContextType<any>) =>
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232251)
```suggestion
const errorCheck = (name: string, formik?: FormikContextType<unknown>) =>
```